Assets/Scenes/base.unity

```diff
@@ -0,0 +1,716 @@
+%YAML 1.1
```

Comment:
I recommend deleting this scene. Computron should live in the PuzzleScene Scene

Reply:
Done. I only had this scene as a placeholder since I wasn't synced up with the version containing PuzzleScene.